src/comm/config-util.js

```diff
@@ -38,8 +38,10 @@ module.exports.setConfigSetting = (name, value) => {
 //
 //Internal method to get an override setting to the configuration settings
 //
-exports.getConfigSetting = (name, default_value) => {
+module.exports.getConfigSetting = (name, default_value) => {
     const config = exports.getConfig();
+    config.set('core:skipStartScript', config._config.argv().get('skipStartScript'));
+    config.set('core:skipEndScript', config._config.argv().get('skipEndScript'));
```

Comment:
Please remove Line 43-44, since they're automatically done by nconf.

Reply:
you mean i should modify the function constructor() of Config class in src/comm/Config.js.  but it also need use set function. 
i am not understand about  "automatically done by nconf"

Reply:
When you execute the following:
`node main.js -c someconfig.yaml -n somenetwork.yaml --core:skipStartScript=false`
nconf will automatically save every argument (and environment variable, and attributes in the config file, etc.) when it's initialized. So you don't need to set `core:skipStartScript` manually. 

That's the beauty of nconf. If you need some new runtime options (like with this PR), you just: 
1. write the code for reading that setting
2. add a default to the config file
3. the user can set it from outside the program (like command line argument, env. variable)

So no need to "parse it" yourself.